project.clj

```diff
@@ -18,14 +18,15 @@
                  [puppetlabs/kitchensink ~ks-version]
                  [puppetlabs/certificate-authority "0.6.0"]
                  [puppetlabs/http-client "0.4.0"]
-                 [org.jruby/jruby-core "1.7.15" :exclusions [com.github.jnr/jffi com.github.jnr/jnr-x86asm]]
+                 [org.jruby/jruby-core "1.7.19"
+                  :exclusions [com.github.jnr/jffi com.github.jnr/jnr-x86asm]]
+                 [com.github.jnr/jffi "1.2.8"]
```

Comment:
From running mvn deps, it looked to me that jruby-core 1.7.19 was still using jffi 1.2.7 and not jffi 1.2.8.  Do we need jffi 1.2.8 specifically, or would it be safer to just stick with jffi 1.2.7 since the jruby packages presumably were tested upstream with it?